modules/axtask/src/task.rs

```diff
@@ -244,6 +273,15 @@ impl TaskInner {
         self.is_idle
     }
 
+    #[inline]
+    pub(crate) fn cpumask(&self) -> CpuMask {
```

Comment:
Return reference for better performance.

Reply:
BUT it's wrapped by a `SpinNoIrq`